### PR TITLE
Fix convox/proxy not being attached to custom network

### LINKED ISSUE #1020: Local proxy does not get attached to correct docker network
`convox start` does not currently pass `--net` parameter when starting the `convox/proxy` containers, which results in the docker failing to link containers. In addition, the failed `docker run` commands for the proxy containers don't get logged, so there's no indication of failure, apart from the proxies being missing.

### PR DESCRIPTION
**Pull request by bladealslayer**
Fixes #1020.

### CODE REVIEW

File: manifest/service.go [hunk 1]
```diff
@@ -82,11 +82,19 @@ func (s *Service) Proxies(app string) []Proxy {
 		if p.External() {
 			name := fmt.Sprintf("%s-%s-proxy-%d", app, s.Name, p.Balancer)
 
+			networkName := ""
```
Comment (ddollar) [inline manifest/service.go]: Maybe a short comment above this logic talking about its intent? Are we always getting the last network defined?

Comment (bladealslayer) [inline manifest/service.go]: Sure, will add that. I used same logic as for launching the service containers themselves: https://github.com/convox/rack/blob/master/manifest/process.go#L56 It picks the last defined network, yes. According to documentation, `convox` supports only single network, no? https://convox.com/docs/docker-compose-file/

**Comment by bladealslayer** [comment]
@ddollar I refactored this a bit to reuse the network name lookup logic. Does it look ok to you?

**Comment by ddollar** [comment]
Looking good. Please rebase when you can. Would it be possible to add some tests around this NetworkName?

**Comment by bladealslayer** [comment]
@ddollar rebased and added tests.

**Pull request closed without merging**